sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -86,19 +86,25 @@ def from_runner_api_parameter(payload, context):
         context.windowing_strategies.get_by_id(payload.value))
 
 
-class DirectRunner(PipelineRunner):
-  """Executes a single pipeline on the local machine."""
-
+def _get_transform_overrides():
   # A list of PTransformOverride objects to be applied before running a pipeline
   # using DirectRunner.
   # Currently this only works for overrides where the input and output types do
   # not change.
-  # For internal SDK use only. This should not be updated by Beam pipeline
-  # authors.
-  _PTRANSFORM_OVERRIDES = []
+  # For internal SDK use only.
+
+  from apache_beam.runners.direct.sdf_direct_runner import ProcessKeyedElementsViaKeyedWorkItemsOverride
+  from apache_beam.runners.sdf_common import SplittableParDoOverride
```

Comment:
<!--new_thread; commit:7549b5c2ebe2ae47af9066eaf97364a27e828ab5; resolved:0-->
Why are these imports here as opposed to at the top?  If they need to be here for dependency reasons, can you add a comment?  Can you confirm that this change from `_PTRANSFORM_OVERRIDES` is only for circular import-avoidance reasons?